gtk4/src/expression.rs

```diff
@@ -0,0 +1,362 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use glib::{translate::*, ToValue};
+use glib::{IsA, Object, Type, Value};
+use std::boxed::Box as Box_;
+
+glib::wrapper! {
+    #[derive(Debug)]
+    pub struct Expression(Shared<ffi::GtkExpression>);
+
+    match fn {
+        ref => |ptr| ffi::gtk_expression_ref(ptr),
+        unref => |ptr| ffi::gtk_expression_unref(ptr),
+    }
+}
+
+impl glib::StaticType for Expression {
+    #[doc(alias = "gtk_expression_get_type")]
+    fn static_type() -> Type {
+        unsafe { from_glib(ffi::gtk_expression_get_type()) }
+    }
+}
+
+impl Expression {
+    #[doc(alias = "gtk_expression_get_value_type")]
+    pub fn get_value_type(&self) -> Type {
+        skip_assert_initialized!();
+        unsafe { from_glib(ffi::gtk_expression_get_value_type(self.to_glib_none().0)) }
+    }
+
+    #[doc(alias = "gtk_expression_is_static")]
+    pub fn is_static(&self) -> bool {
+        skip_assert_initialized!();
+        unsafe { from_glib(ffi::gtk_expression_is_static(self.to_glib_none().0)) }
+    }
+
+    #[doc(alias = "gtk_expression_bind")]
+    pub fn bind<T: IsA<Object>>(
+        &self,
+        target: &T,
+        property_name: &str,
+        this: Option<&T>,
+    ) -> ExpressionWatch {
+        assert_initialized_main_thread!();
+        unsafe {
+            let ret = ffi::gtk_expression_bind(
+                self.to_glib_none().0,
```

Comment:
```suggestion
                self.to_glib_full(),
```

Reply:
Please also check if there are other `self` parameters that are `transfer full` in this API

Reply:
It was the only one, thanks!